src/main/kotlin/eu/kevin/api/models/ErrorCode.kt

```diff
@@ -1,10 +1,12 @@
 package eu.kevin.api.models
 
+import eu.kevin.api.serializers.enums.ErrorCodeSerializer
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
-@Serializable
+@Serializable(with = ErrorCodeSerializer::class)
 enum class ErrorCode(val code: Int) {
+    @SerialName("-1") UNKNOWN_VALUE(-1),
```

Comment:
There is no need to include the @SerialName annotation for UNKNOWN_VALUE enums

Reply:
I did it only for consistency in each enum. Should I remove it? I was wondering about additional comment above as well for clarity

Reply:
If by some reason "-1" value was returned as error code, it would be parsed to UNKNOWN_VALUE, so that is not a desired behavior. I think we should remove them. Also I would put UNKNOWN_VALUE as the last enum value in each enum, perhaps additional comment is not neccessary